docs/pages/versions/unversioned/sdk/screen-capture.mdx

```diff
@@ -128,6 +128,46 @@ export default function useScreenCaptureCallback() {
 
 </SnackInline>
 
+## Screen Recording Events
+
+Screen recording detection is available on iOS 11 and newer. The API lets you detect when users start or stop recording your app's screen, allowing you to protect sensitive content.
+
+#### Example
+
+```javascript
+import * as ScreenCapture from 'expo-screen-capture';
+import { useEffect } from 'react';
```

Comment:
```suggestion
import { useEffect, StyleSheet } from 'react';
```

Let's change how styles are used in the example to follow the same pattern from other examples in this doc.